integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -50,6 +51,8 @@ COPY --from=ti /opt/ti/sysconfig_1.13.0 /opt/ti/sysconfig_1.13.0
 COPY --from=openiotsdk /opt/gcc-arm-none-eabi-10.3-2021.10/ /opt/gcc-arm-none-eabi-10.3-2021.10/
 COPY --from=openiotsdk /opt/FVP_Corstone_SSE-300/ /opt/FVP_Corstone_SSE-300/
 
+COPY --from=bouffalolab ~/.bouffalolab ~/.bouffalolab
```

Comment:
why in $HOME? please use something like /opt like all the other toolchains.

Reply:
Updated as your suggestion. Put it under /opt/bouffalolab_sdk.